packages/arb-ts/integration_test/sanity.test.ts

```diff
@@ -0,0 +1,216 @@
+import { expect } from 'chai'
+import { L1ERC20Gateway__factory } from '../src/lib/abi/factories/L1ERC20Gateway__factory'
+import { L2ERC20Gateway__factory } from '../src/lib/abi/factories/L2ERC20Gateway__factory'
+import { L1CustomGateway__factory } from '../src/lib/abi/factories/L1CustomGateway__factory'
+import { L2CustomGateway__factory } from '../src/lib/abi/factories/L2CustomGateway__factory'
+import { L1WethGateway__factory } from '../src/lib/abi/factories/L1WethGateway__factory'
+import { L2WethGateway__factory } from '../src/lib/abi/factories/L2WethGateway__factory'
+import { L2GatewayRouter__factory } from '../src/lib/abi/factories/L2GatewayRouter__factory'
+import { L1GatewayRouter__factory } from '../src/lib/abi/factories/L1GatewayRouter__factory'
+
+import { TestCustomTokenL1__factory } from '../src/lib/abi/factories/TestCustomTokenL1__factory'
+import { IArbToken__factory } from '../src/lib/abi/factories/IArbToken__factory'
+
+import { AeWETH__factory } from '../src/lib/abi/factories/AeWETH__factory'
+
+import { instantiateBridge } from '../scripts/instantiate_bridge'
+import { existentTestERC20, existentTestCustomToken } from './testHelpers'
+import { constants } from 'ethers'
+
+describe('sanity checks (read-only)', async () => {
+  it('standard gateways public storage vars properly set', async () => {
+    const { bridge, l1Network, l2Network } = await instantiateBridge()
```

Comment:
any reason to not instantiate this once in a before hook instead of each test case? here we don't use a random wallet in each test